public/resources/img/icon/touch-emulator-off.svg

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
Please run this SVG through [svgo](https://www.npmjs.com/package/svgo) so that all the "bits that do nothing", comments, nonstandard attributes, etc. get removed